packages/block-editor/src/components/global-styles/filters-panel.js

```diff
@@ -125,6 +162,7 @@ export default function FiltersPanel( {
 	settings,
 	panelId,
 	defaultControls = DEFAULT_CONTROLS,
+	clearable = false,
```

Comment:
This component is private and I don't think there is a need to make `clearable` prop mutable at this time.